test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.2
+2.3
```

Comment:
bump to 2.5 ref: https://github.com/kubernetes/kubernetes/pull/79423#issuecomment-510526158